docs/source/UserGuide/GettingStarted/InstallingClient/index.rst

```diff
@@ -25,19 +25,19 @@ Download the *dirac-install* script from::
 
 Choose the directory where you want to install the DIRAC software and run the dirac-install script from
 this directory giving the appropriate version of the DIRAC release,
-and, the version of the "lcgBundle" (with "-g" option) that you want to use::
+and the version of the "lcgBundle" (with "-g" option), to be skipped by MacOS users due to its current unavailability, that you want to use::
```

Comment:
This is not a macOS specific problems, in many cases errors will also occur on linux that are not RedHat 6 or 7 equivalent.